src/modules/Security.rb

```diff
@@ -444,6 +448,16 @@ def read_polkit_settings
         "#{@Settings['HIBERNATE_SYSTEM']}"
     end
 
+    def read_selinux_settings
+      @Settings.delete("SELINUX_MODE")
```

Comment:
@schubi2, I didn't catch the idea of using the `@Settings["SELINUX_MODE"]` but I was not brave enough for getting rid of it after adding the convenience method `#selinux_config`, which keep a single instance of `Y2Security::Selinux`. So, if this settings is there just to hold the SELinux mode, I think it can be removed, which would allow to simplify the code.

Reply:
Nevermind :) I got it after a few manual tests and also reading https://yastgithubio.readthedocs.io/en/latest/autoyast-development/#autoyast-control-files-old-style 

It's needed to export the value, for example when cloning the system. So, I have only to update the #write_selinux method according to https://github.com/yast/yast-security/pull/88#discussion_r575179956 and maybe exporting it also when is disabled instead of omitting it.

Let's think a little bit more about it.